solarsystem.rb

```diff
@@ -0,0 +1,203 @@
+class Planet
+  attr_accessor :name, :moons, :orbit, :distance_from_sun
+  # initializes attributes of each instance of a planet
+def initialize(planet_hash)
+    @name = planet_hash[:name]
+    @moons = planet_hash[:moons]
+    @orbit = planet_hash[:orbit]
+    @distance_from_sun = planet_hash[:distance_from_sun]
+  end
+
+  # prints information about a specific planet
+def print_info
+  puts "Here are some basic facts about #{@name}: It has #{@moons} moon(s),\norbits around the sun in #{@orbit} Earth days, and is #{@distance_from_sun}\n million miles away from the Sun."
+end
+
+def distance_from(planet2)
+  (@distance_from_sun - planet2.distance_from_sun).abs
+end
+
+def add_planets(planet)
+    planet.each do |planet|
+    @planets << planet
+    end #ends do loop
+  end #ends add planets
```

Comment:
This loop looks a little confusing because you're looping through `planet` and calling each individual item in the array, `planet`.  I would suggest naming the parameter `planets` to reduce confusion.